### PR TITLE
app: l10n: Update Indonesian translations

### PR DESCRIPTION
**Pull request by fadlyas07**
It will be easier and more comfortable to read :)

### CODE REVIEW

File: app/src/main/res/values-in/strings.xml [hunk 1]
```diff
@@ -171,35 +190,51 @@
...
+    <string name="download">Unduh</string>
+    <string name="reboot">Mulai ulang</string>
     <string name="release_notes">Catatan rilis</string>
     <string name="flashing">Memasang…</string>
     <string name="done">Selesai!</string>
     <string name="failure">Gagal!</string>
+    <string name="hide_app_title">Menyembunyikan aplikasi Magisk…</string>
     <string name="open_link_failed_toast">Tidak ditemukan aplikasi untuk membuka link ini</string>
-    <string name="complete_uninstall">Uninstal penuh</string>
+    <string name="complete_uninstall">Pencopotan selesai</string>
```
[the remark this answers is not in the document]
Comment (fadlyas07) [reply app/src/main/res/values-in/strings.xml:214]: oh ah wkwk ok, but what do you think about the meaning of "daemon" in bahasa? is there the right word to define it?

[the remark this answers is not in the document]
Comment (Stillhard) [reply app/src/main/res/values-in/strings.xml:214]: Lol, simple, don't, we wouldn't want to read it as 1. Setan nor 2. Iblis as the real meaning wouldn't we?

[the remark this answers is not in the document]
Comment (fadlyas07) [reply app/src/main/res/values-in/strings.xml:214]: ah okk i think maybe you have some interesting words to mean it haha

[the remark this answers is not in the document]
Comment (fadlyas07) [reply app/src/main/res/values-in/strings.xml:214]: well thanks btw, i'm gonna making a new pull request later :)

**Pull request closed without merging**